nss-day-cohort-39/tabloid-full-stack-velours
# Tag CRUD Implementation

### Changes
- Full tag CRUD implemented. Back-end controller, model, repo and front-end provider and react implementation. 

### Testing
- Confirm "Tags" link appears in menu
- Confirm all tag are listed and sorted alphabetically
- Add a new tag by typing in the name in the input and pressing save
- Edit a new or existing tag by clicking the pencil. Change the name (or don't) and press Save.
- Delete a tag
- Confirm the styling is cohesive between tags and categories

Resolves #13 , Resolves #14, Resolves #15, Resolves #16
